meteor/client/ui/RundownView/RundownTiming/RundownTimingProvider.tsx

```diff
@@ -121,6 +121,9 @@ export const RundownTimingProvider = withTracker<
 		private temporaryPartInstances: Map<PartId, PartInstance> = new Map<PartId, PartInstance>()
 
 		private linearParts: Array<[PartId, number | null]> = []
+		private prevPartId: string | null = null
```

Comment:
Some sort of a comment about what this value will be used for would be good, since "prevPartId" could mean a lot of things.